api/v1alpha1/gatekeeper_types.go

```diff
@@ -92,6 +92,8 @@ type AuditConfig struct {
 	// +optional
 	ConstraintViolationLimit *uint64 `json:"constraintViolationLimit,omitempty"`
 	// +optional
+	// Setting Automatic lets the Gatekeeper operator manage syncOnly in the config resource
+	// NOTE: We don't recommend to use Automatic when you use referential constraint
```

Comment:
```suggestion
	// Setting Automatic lets the Gatekeeper operator manage syncOnly in the config resource.
	// It is not recommended to use Automatic when you use referential constraints since those are not detected.
```